Configurations.md

```diff
@@ -2481,6 +2481,34 @@ fn main() {
 
 See also [`max_width`](#max_width) and [`use_small_heuristics`](#use_small_heuristics)
 
+## `single_line_simple_if`
+
+Allows simple single expression if blocks to format on one line. Useful in the case of keeping `let-else` guards format consistent with `if` guards.
+
+Note that line will still break if:
+1. The condition length is over the `max_width`/`single_line_simple_if_max_width length`
+2. The block contains a trailing semicolon
+3. The block contains a single or multi-lined comment
+
+- **Default value**: `false`
+- **Possible values**: `true`, `false`
+- **Stable**: No
+
+#### `false` (default):
+
+## `single_line_simple_if_max_width`
+
+Maximum line length for single line if with a simple inner expression. Useful in the case of keeping `let-else` guards format consistent with `if` guards.
+
+A value of `0` (zero) results in if-else expressions always being broken into multiple lines. Note this occurs when `use_small_heuristics` is set to `Off`.
+
+- **Default value**: `50`
+- **Possible values**: any positive integer that is less than or equal to the value specified for [`max_width`](#max_width)
+- **Stable**: Yes
+
+By default this option is set as a percentage of [`max_width`](#max_width) provided by [`use_small_heuristics`](#use_small_heuristics), but a value set directly for `single_line_if_else_max_width` will take precedence.
```

Comment:
let me think on whether we need a separate width configuration for this. Originally I was thinking we'd reuse the value for `single_line_if_else_max_width`.

Reply:
Ya we probably should. Currently it seems `single_line_if_else_max_width` is for expressions only, more specifically `ExprType::SubExpression` in rustfmt's code, not plain if or if-else blocks. The `single_line_if_else_max_width` in only applying when there is a semicolon at the end of the line, so only if the `if` is setting a value with `let if` or is inside a function parameter. I'll have some examples in the test files next push to see what I mean if that doesn't make sense.